src/test/java/edu/hw1/task8/Task8Test.java

```diff
@@ -0,0 +1,50 @@
+package edu.hw1.task8;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class Task8Test {
+    Task8 task8;
+
+    @BeforeEach
+    void setUp() {
+        task8 = new Task8();
+    }
+
+    @Test
+    void checkIfTheyAreSafe() {
```

Comment:
`BUG`  в джаве методы тестов принято называть в snake_case
```suggestion
    void check_if_they_are_safe() {
```